src/components/ExpenseForm/ExpenseForm.jsx

```diff
@@ -1,23 +1,49 @@
 import React, { useState, useEffect } from 'react'
 import { DataService } from '../../services/DataService'
 import DatePicker from 'react-datepicker'
-import { expensesUrl } from '../../static/apiconfig'
+import { EXPENSES_URL } from '../../static/apiconfig'
+import ChooseCategory from '../ChooseCategory/ChooseCategory'
+import ChooseSubcategory from '../ChooseSubcategory/ChooseSubcategory'
+import { CATEGORIES_URL } from '../../static/apiconfig'
```

Comment:
you could just:
`import { CATEGORIES_URL, EXPENSES_URL } from '../../static/apiconfig'`

Reply:
Fixed